dma_frontend/src/components/SignInComp.jsx

```diff
@@ -1,11 +1,69 @@
 import React from 'react';
-import '../styles/login.css';
-
+import '../styles/signin.css';
+import signin from '../images/signin.jpg'
 const SignInComp = () => {
   return (
     <div className=''>
       <div className=''>
-        {/* TYPE YOUR CODE HERE. IF YOU USE TAILWINDCSS MAKE SURE TO ADD THE tw- PREFIX TO THE UTILITY CLASS AND IF YOU WRITE VANILLA CSS KINDLY USE login.css FILE */}
+        {         <div class="in-container">
+         <div class="in-item-2">
```

Comment:
Please check through this whole file and change any html entity that has `class` to `className`.  

I only highlighted this section as an example for you. please look through the whole file and change anywhere this kind of issue appears